examples/communication/Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[workspace]
+members = [
+    "*",
+]
+exclude = [
+    "styling",
+    "target",
+    ".cargo"
+]
+resolver = "2"
```

Comment:
Since nested workspaces are [not a thing yet](https://github.com/rust-lang/cargo/issues/5042), I'd like to avoid making this into yet another workspace. All the examples should live inside one workspace - so the CI and other things can more easily find them.

Reply:
The communication examples have been restructured as top-level examples.